scripts/report/_01_make_reports.py

```diff
@@ -652,58 +665,60 @@ def run_report_source(
 
     conditions.extend(cfg.contrasts)
 
-    if cfg.use_template_mri:
-        fname_trans = 'fsaverage'
-        has_trans = True
-    else:
-        fname_trans = bids_path.copy().update(suffix='trans')
-        has_trans = fname_trans.fpath.exists()
```

Comment:
Not necessary, we create a custom-named trans file even when using the template MRI.